massa-models/src/block.rs

```diff
@@ -126,7 +125,7 @@ pub struct Block {
     /// signed header
     pub header: WrappedHeader,
     /// operations
-    pub operations: Vec<WrappedOperation>,
+    pub operations: OperationIds,
```

Comment:
Critical: operation order in blocks matters, but OperationIds is a Set.

* operation order is chosen by block producer
* operations are executed in the order they appear in the block
* the operation global hash in the header is sensitive to operation order
* it is therefore consensus-critical for the operations order to be the same for everyone for every block

```suggestion
    pub operations: Vec<OperationId>,
```

Reply:
Done.